src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -957,10 +953,14 @@ private String getFindByIndexBody(String selectorJson,
         return finalbody.toString();
     }
 
-    HttpResponse executeRequest(HttpRequestBase request) {
+    InputStream executeRequest(HttpConnection request) {
```

Comment:
This method should probably be removed, it seems like most places call `client.executeRequest()` anyway and if they don't they probably should.